.env.template

```
@@ -27,4 +27,7 @@ NEXT_PUBLIC_POSTHOG_HOST=
 
 # Formbricks Configuration
 NEXT_PUBLIC_FORMBRICKS_ENVIRONMENT_ID=
-NEXT_PUBLIC_FORMBRICKS_API_HOST=
\ No newline at end of file
+NEXT_PUBLIC_FORMBRICKS_API_HOST=
+
+# Multimodal Features
+NEXT_PUBLIC_MULTIMODAL_ENABLED=disabled
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)